src/modules/scraper/service.ts

```diff
@@ -253,6 +253,10 @@ export class ScraperService {
       return 15;
     }
 
+    if (chainId === ChainIds.cher) {
+      return 15;
+    }
+
```

Comment:
There looks to be a pattern here that OP chains follow this 15 block lookback. We should probably optimize this